dvc/command/add.py

```diff
@@ -26,7 +26,8 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    ADD_HELP = "Add files/directories to dvc."
+    ADD_HELP = "Add files/directories to dvc. \n\
```

Comment:
Let's reformat all of these this way:
```
    ADD_HELP = (
        "Add files/directories to dvc. \n"
        "documentation: https://man.dvc.org/add"
    )
```

so it is unified everywhere and we don't mix \ into all of these.

Reply:
Ok done